index.css

```diff
@@ -0,0 +1,304 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+.header1 {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: auto;
+  padding: 30px 0;
+}
+
+.header1 h3 {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.header1 h3 a {
+  text-decoration: none;
+}
+
+.section1 {
+  background: url(./img//background1.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 90%;
+  margin: auto;
+  padding: 100px 0;
+}
```

Comment:
Your project design looks great, I commend you. As the previous reviewer requested, kindly adjust the background to better match the design template. As you might notice, the images that constitute the background help to use up the white blank space on the page and will look better.

Some of the changes you can make to your css include:

- removing the width defined
- Adding left and right padding
- setting background-size property to 100%

e.g: 
```
.section1 {
    background: url(./img//background1.png);
    background-size: 100%;
    background-repeat: no-repeat;
    margin: auto;
    padding: 5%;
}
```
You might also need to remove the width property assigned to `.section1 p` element

To avoid an overlap between the h1 text and the background, you might need to give some margin-top to `.section1 h1` element

These are only suggestions, however you wish to achieve this is entirely up to you.


Your design:
![image](https://user-images.githubusercontent.com/20554583/205246345-c7d9803c-a643-4e9f-a407-d583cb6f148d.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/205246471-50007af0-80b6-4c6e-a128-9bdd6bdd987c.png)